test/index.ts

```diff
@@ -559,58 +564,67 @@ describe('PubSub', () => {
       const apiResponse = {};
 
       beforeEach(() => {
-        pubsub.request = (config, callback) => {
-          callback(error, apiResponse);
-        };
+        pubsub.request =
+            (config: pubsubTypes.RequestConfig, callback: Function) => {
+              callback(error, apiResponse);
+            };
       });
 
       it('should return an error & API response', done => {
-        pubsub.createTopic('new-topic', (err, topic, apiResponse_) => {
-          assert.strictEqual(err, error);
-          assert.strictEqual(topic, null);
-          assert.strictEqual(apiResponse_, apiResponse);
-          done();
-        });
+        pubsub.createTopic(
+            'new-topic',
+            (err?: Error|null, topic?: pubsubTypes.Topic|null,
+             apiResponse_?: google.pubsub.v1.ITopic) => {
+              assert.strictEqual(err, error);
+              assert.strictEqual(topic, null);
+              assert.strictEqual(apiResponse_, apiResponse);
+              done();
+            });
       });
     });
 
     describe('success', () => {
       const apiResponse = {};
 
       beforeEach(() => {
-        pubsub.request = (config, callback) => {
-          callback(null, apiResponse);
-        };
+        pubsub.request =
+            (config: pubsubTypes.RequestConfig, callback: Function) => {
+              callback(null, apiResponse);
+            };
       });
 
       it('should return a Topic object', done => {
         const topicName = 'new-topic';
         const topicInstance = {};
 
-        pubsub.topic = name => {
+        pubsub.topic = (name: string) => {
           assert.strictEqual(name, topicName);
-          return topicInstance;
+          return topicInstance as pubsubTypes.Topic;
         };
 
-        pubsub.createTopic(topicName, (err, topic) => {
-          assert.ifError(err);
-          assert.strictEqual(topic, topicInstance);
-          done();
-        });
+        pubsub.createTopic(
+            topicName, (err?: Error|null, topic?: pubsubTypes.Topic|null) => {
```

Comment:
I'm surprised we need this, shouldn't the types be inferred?

Reply:
agree, here no need of declaring type, it is by default. 
![image](https://user-images.githubusercontent.com/44371467/53551389-3a721e00-3b5f-11e9-8a74-fda522ccfac3.png)
in this way its look more informative

Reply:
No, please don't include the types here unless you have to.  Let the compiler do the work!